src/kbc_tools.py

```diff
@@ -134,3 +138,17 @@ def result_iterator():
             for future in buffer:
                 future.cancel()
     return result_iterator()
+
+
+def serialize_data(obj, compress=True):
+    bin_data = pickle.dumps(obj)
+    if compress:
+        bin_data = bz2.compress(bin_data)
+    return base64.encodebytes(bin_data).decode('ascii')
```

Comment:
I would say serialization should return `bin_data` and encoding to base64 should be done separately. But since this is a single use function, it doesn't matter that much.

Reply:
I just wanted to have this functionality separate from the main code. And since we work with CSV files, we need a textual value not bytes.